docs/contents.rst.inc

```diff
@@ -92,6 +92,7 @@ Help yourself
    :caption: What to do if things go wrong
 
    basics/101-135-help
+   basics/101-136-cheatsheet
```

Comment:
what about also adding a reference to the cheat sheet really high on the leading page, e.g. right before **Introduction** with something like

`DataLad Cheatsheet <>`_ provides a quick overview of DataLad principles and common commands for impatient.

Reply:
yeah, that makes a lot of sense. :+1: